tests/service/Symbols.fs

```diff
@@ -474,207 +474,3 @@ type Foo =
           (:? FSharpMemberOrFunctionOrValue as setMfv) ->
             Assert.AreNotEqual(getMfv.CurriedParameterGroups, setMfv.CurriedParameterGroups)
         | _ -> Assert.Fail "Expected symbols to be FSharpMemberOrFunctionOrValue"
-        
-    [<Test>]
```

Comment:
@KevinRansom why did you remove these tests?
You really removed a bunch of these that are not even related!

Reply:
@nojaf it was just plain revert of the following commit:
https://github.com/dotnet/fsharp/commit/2ef33c4edf540805d78e591b083d7da9396c996a

Reply:
Or maybe not?

Reply:
Clearly not: https://github.com/dotnet/fsharp/commit/2ef33c4edf540805d78e591b083d7da9396c996a#diff-86de212e8dc5b638058239f2b2b485cd6207f61b3acb2556de8bb7fef15065ea

Tests are removed that were not introduced by https://github.com/dotnet/fsharp/pull/15214

Reply:
Seems mistakenly removed. I will re-add them

Reply:
Thanks